safenode/src/client/wallet.rs

```diff
@@ -0,0 +1,50 @@
+// Copyright 2023 MaidSafe.net limited.
+//
+// This SAFE Network Software is licensed to you under The General Public License (GPL), version 3.
+// Unless required by applicable law or agreed to in writing, the SAFE Network Software distributed
+// under the GPL Licence is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. Please review the Licences for the specific language governing
+// permissions and limitations relating to use of the SAFE Network Software.
+
+use sn_dbc::{Dbc, DbcIdSource, DerivedKey, PublicAddress, Token};
+
+use crate::protocol::{
+    offline_transfers::{create_transfer, Outputs as TransferDetails},
+    wallet::{Result, SendClient, SendWallet},
+};
+
+use super::Client;
+
+/// A wallet client can be used to send and
+/// receive tokens to/from other wallets.
+pub struct WalletClient<W: SendWallet> {
+    client: Client,
+    wallet: W,
+}
+
+impl<W: SendWallet> WalletClient<W> {
+    /// Create a new wallet client.
+    pub fn new(client: Client, wallet: W) -> Self {
+        Self { client, wallet }
+    }
+
+    /// Send tokens to another wallet.
+    pub async fn send(&mut self, amount: Token, to: PublicAddress) -> Result<()> {
```

Comment:
After sending, usually it is not bad to print transaction ID or some information to be track by user. Here we don't return anything. Is it OK?

Reply:
Yeah, maybe so.

My vision here is rather that a UI uses the api to get the data to show in the UI, rather than relying on log lines.

Reply:
> Tx ids is way too low level for a user.

Probably some users might be interested to check the id. In case of any issue on the production, This information probably are so valuable that time.

Reply:
Yeah, you're right, some log lines can be useful.